spec/readthis/cache_spec.rb

```diff
@@ -144,6 +144,10 @@
         'e' => 2,
       )
     end
+
+    it 'returns {} with no keys' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.